lib/zuora/objects/base.rb

```diff
@@ -140,6 +144,7 @@ def update
 
     # destroy the remote object
     def destroy
```

Comment:
They are copied from ActiveRecord. I added a comment that describes why these methods are needed.